# 1.3 At

## Decay properties   

| nuclide | $T_{1/2}$ |  error |
| ------------------|-----------------------------|----------------|
|$^{191}$At | 2.1   ms | 0.8 |
|$^{192}$At | 11.5   ms | 0.6 |
|$^{193}$At | 29   ms | 5 |

## paper

   - [$^{191,193,195}$At](../paper/At.pdf)
   - [$^{199,205,207}$At](../paper/VASSILISSA.pdf)

## Cross section 

   
|N0|                Reaction             | $C_{f}$(cal) | $C_{f}$(Ref) |
|:----:| :---------------------------------: |  :---: |:-------:|
|1.|$^{26}$Mg + $^{181}$Ta  →  $^{207}$At$^{\star}$ |0.80|   ${\star}$   |
|2.|$^{24}$Mg + $^{181}$Ta  →  $^{205}$At$^{\star}$ |0.79|   ${\star}$   |
|3.|$^{40}$Ar + $^{165}$Ho  →  $^{205}$At$^{\star}$ |0.79|   ${\star}$   |
|4.|$^{40}$Ca + $^{159}$Tb  →  $^{199}$At$^{\star}$ |0.73|   ${\star}$   |
|5.|$^{56}$Fe + $^{141}$Pr  →  $^{197}$At$^{\star}$ |0.66|   ${\star}$   |
|6.|$^{54}$Fe + $^{141}$Pr  →  $^{195}$At$^{\star}$ |0.59|   ${\star}$   |
|7.|$^{51}$V + $^{144}$Sm  →  $^{195}$At$^{\star}$ |0.59|   ${\star}$   |

In [1]:
%jsroot on

   - 1.  $^{26}$Mg + $^{181}$Ta  →  $^{207}$At$^{\star}$
   
|Excitation energy  | channel| product | cross section (error = ${\pm}50\% $ )      | 
|:---:|:----------:|:----------:|:-------------------:|
|${\star}$| 4n| $^{203}$At |${\sigma}_{max} = 14 mb$|
|${\star}$| 5n| $^{202}$At |${\sigma}_{max} = 40  mb$|
|${\star}$| 6n| $^{201}$At |${\sigma}_{max} = 20 mb$|
|${\star}$| 7n| $^{200}$At |${\sigma}_{max} = 8 mb$|

In [2]:
gROOT->ProcessLine(".L ../data.cpp");
data("Mg^{26}+Ta^{181}", "At", 1, 11, 11, 0.80, 1.0E-03, 100000.0E1, 50.31, 110.0);

   - 2.  $^{24}$Mg + $^{181}$Ta  →  $^{205}$At$^{\star}$
   
|Excitation energy  | channel| product | cross section (error = ${\pm}50\% $ )      | 
|:---:|:----------:|:----------:|:-------------------:|
|${\star}$| 5n| $^{200}$At |${\sigma}_{max} = 30 mb$|
|${\star}$| 6n| $^{199}$At |${\sigma}_{max} = 4.5  mb$|
|${\star}$| 7n| $^{198}$At |${\sigma}_{max} = 850 \mu b$|
|${\star}$| 8n| $^{197}$At |${\sigma}_{max} = 54 \mu b$|

In [3]:
gROOT->ProcessLine(".L ../data.cpp");
data("Mg^{24}+Ta^{181}", "At", 1, 11, 11, 0.80, 1.0E-03, 10000.0E1, 53.76, 110.0);

   - 3.  $^{40}$Ar + $^{165}$Ho  →  $^{205}$At$^{\star}$
   
|Excitation energy  | channel| product | cross section (error = ${\pm}50\% $ )      | 
|:---:|:----------:|:----------:|:-------------------:|
|${\star}$| 5n| $^{200}$At |${\sigma}_{max} = 11.5 mb$|
|${\star}$| 6n| $^{199}$At |${\sigma}_{max} = 2  mb$|
|${\star}$| 7n| $^{198}$At |${\sigma}_{max} = 400 \mu b$|
|${\star}$| 8n| $^{197}$At |${\sigma}_{max} = 35 \mu b$|
|${\star}$| 9n| $^{196}$At |${\sigma}_{max} = 3.5 \mu b$|

In [4]:
gROOT->ProcessLine(".L ../data.cpp");
data("Ar^{40}+Ho^{165}", "At", 1, 11, 11, 0.80, 1.0E-03, 10000.0E1, 50.97, 110.0);

   - 4.  $^{40}$Ca + $^{159}$Tb  →  $^{199}$At$^{\star}$
   
|Excitation energy  | channel| product | cross section (error = ${\pm}50\% $ )      | 
|:---:|:----------:|:----------:|:-------------------:|
|${\star}$| 3n| $^{196}$At |${\sigma}_{max} = 3.6 \mu b$|

In [5]:
gROOT->ProcessLine(".L ../data.cpp");
data("Ca^{40}+Tb^{159}", "At", 1, 11, 11, 0.66, 1.0E-06, 1.0E1, 53.90, 100.0);

   - 5.  $^{56}$Fe + $^{141}$Pr  →  $^{197}$At$^{\star}$  －(4n)→  $^{193}$At
   
|Beam energy  |  cross section | 
|:---:|:----------:| 
|  266(50.2) MeV|    40 nb      |

In [6]:
gROOT->ProcessLine(".L ../data.cpp");
data("Fe^{56}+Pr^{141}", "At", 0, 11, 11, 0.66, 1.0E-06, 1.0E1, 33.97, 80.0);

   - 6.  $^{54}$Fe + $^{141}$Pr  →  $^{195}$At$^{\star}$  －(4n)→  $^{191}$At
   
|Beam energy  |  cross section | 
|:---:|:----------:| 
|  260(49.2) MeV|    300 pb      |

In [3]:
gROOT->ProcessLine(".L ../data.cpp");
data("Fe^{54}+Pr^{141}", "At", 0, 11, 11, 0.59, 1.0E-07, 2.0E-1, 36.5, 60.0);

Warning in <TCanvas::Constructor>: Deleting canvas with same name: Fe^{54}+Pr^{141}->At


   - 7.  $^{51}$V + $^{144}$Sm  →  $^{195}$At$^{\star}$  －(3n)→  $^{192}$At
   
|Excitation energy|Beam energy  |  cross section | 
|:---:|:---:|:----------:| 
|39.2MeV|  230(1) MeV|    300 pb      |

In [7]:
gROOT->ProcessLine(".L ../data.cpp");
data("^{51}V+^{144}Sm", "At", 2, 11, 11, 0.59, 1.0E-07, 1.0E0, 32.8, 55.0);

Warning in <TCanvas::Constructor>: Deleting canvas with same name: ^{51}V+^{144}Sm->At


In [8]:
Double_t A[]={195, 197, 199, 205, 207}; //
Double_t Cf[] ={0.59, 0.66, 0.73, 0.79, 0.8}; 
Double_t N[5];
for(int i=0; i<5; i++)
    N[i] = A[i] - 85;

In [9]:
TGraph *AC = new TGraph(5, A, Cf);
auto canvas = new TCanvas("Cf", "Cf");

In [10]:
AC->SetMarkerStyle(20);
AC->SetTitle("Cf-A(Po); A; Cf");
AC->Draw("AP");
AC->Fit("pol1");
canvas->Draw();


****************************************
Minimizer is Linear / Migrad
Chi2                      =   0.00321716
NDf                       =            3
p0                        =     -2.55698   +/-   0.634637    
p1                        =     0.016306   +/-   0.00316285  


In [11]:
TGraph *NC = new TGraph(5, N, Cf);
NC->SetMarkerStyle(20);
NC->SetTitle("Cf-N(Po); N; Cf");
NC->Draw("AP");
NC->Fit("pol1");
canvas->Draw();


****************************************
Minimizer is Linear / Migrad
Chi2                      =   0.00321716
NDf                       =            3
p0                        =     -1.17097   +/-   0.365919    
p1                        =     0.016306   +/-   0.00316285  


***<mark style=background-color:Blue><font size=5 face="微软雅黑" color="Red" >Unknow</font><mark>***

# $^{193}At$

In [12]:
cout << "Cf = " << AC->Eval(193);

Cf = 0.52

## $^{78}$Kr + $^{115}$In  →  $^{193}$At$^{\star}$  

In [13]:
//gROOT->ProcessLine(".L ../data.cpp");
data("Kr^{78}+In^{115}", "At", 0, 11, 11, 0.52, 1.0E-07, 5.0E-4, 35.8, 60.0);

In [5]:
gROOT->ProcessLine(".L ../data.cpp");
data("^{78}Kr+^{115}In", "all", 0, 24, 24, 0.52, 1.0E-10, 1.0E-1, 35.8, 80.0, 1, 40.0, 44.0);

190At  0.000852
191At  2.674e-05
189Po  1.28763e-05
190Po  0.00206
191Po  0.000383
187Bi  0.01281
188Bi  0.007658
189Bi  0.0019224
190Bi  0.118
191Bi  0.00273
186Pb  0.000968
187Pb  0.1479
188Pb  0.00902
189Pb  0.011267
190Pb  0.0616


Warning in <TCanvas::Constructor>: Deleting canvas with same name: ^{78}Kr+^{115}In->all


<font size=4 color="Red" face="微软雅黑">

   - 40~46 MeV---$^{190}At$

| Prodection| cross section($\mu$ b) |  decay |
| ------------------|-----------------------------|----------------|
|$^{187}Pb$|1.48e-1|$\alpha$/EC(88\90%) 15.2/18.3s|
|$^{190}Bi$|1.18e-1|$\alpha$(90\70%)/EC 6.3/6.2s|
|$^{190}Pb$|6.16e-2|$\alpha$/EC(99.6%) 71s|
|$^{187}Bi$|1.28e-2|$\alpha$ 37/0.37 ms|
|$^{189}Pb$|1.13e-2|EC 39/50s|
|$^{188}Pb$|9.02e-3|$\alpha$/EC(91%) 25.1s|
|$^{188}Bi$|7.66e-3|$\alpha$ 265/60 ms|
|$^{191}Bi$|2.73e-3|$\alpha(51/68)$/EC(49/)/IT(/32) 12.5/0.125s|
|$^{190}Po$|2.06e-3|$\alpha$ 2.46ms|
|$^{189}Bi$|1.92e-3|$\alpha$(>50%)/EC 674/5ms|
|$^{186}Pb$|9.68e-4|$\alpha$/EC(60%) 4.82s|
|<font color="Red"> $^{190}At$</font>|8.52e-4|$\alpha $61.4 ms#| 
|$^{191}Po$|3.83e-4|$\alpha$(99%) 22/93ms|
|$^{191}At$|2.67e-5|$\alpha$ 1.7/2.1ms|
|$^{189}Po$|1.29e-5|$\alpha$ 3.5ms|
</font>

## $^{90}$Zr + $^{103}$Rh  →  $^{193}$At$^{\star}$  (Cf = 0.52) 

<font size=4 color="Red" face="微软雅黑"> 
   - 同样的弹靶组合下，$1pxn$反应道要比$xn$道的截面要大，基本上相差一个量级 
   - 同一蒸发余核，$1pxn$反应道要比$xn$道的截面要小，基本上相差三个量级

# $^{191}At$

In [14]:
cout << "Cf = " << AC->Eval(191);

Cf = 0.45

## $^{78}$Kr + $^{113}$In  →  $^{191}$At$^{\star}$  

In [15]:
//gROOT->ProcessLine(".L ../data.cpp");
data("Kr^{78}+In^{113}", "At", 0, 11, 11, 0.52, 1.0E-10, 5.0E-4, 31.81, 60.0);

***<mark style=background-color:Blue><font size=5 face="微软雅黑" color="Red" >Conclusion</font><mark>***

***<mark style=background-color:Blue><font size=5 face="微软雅黑" color="Red" >New isotopes  $^{189、190}At$</font><mark>***

<font size=4 color="Red" face="微软雅黑">
    
| Projection & target| Production |  Cross sectoion |
| ------------------|-----------------------------|----------------|
|$^{58}_{28}$Ni + $^{136}_{58}$Ce | 1p3n → $^{190}_{85}$At | ${\sigma}_{max}$= 20 pb |
|$^{78}_{36}$Kr + $^{114}_{50}$Sn | 1p2n → $^{189}_{85}$At| ${\sigma}_{max}$ = 3 pb |
|$^{78}_{36}$Kr + $^{114}_{50}$Sn | 1p1n → $^{190}_{85}$At| ${\sigma}_{max}$= 20 pb |
|<font color="Red"> $^{78}_{36}$Kr + $^{113}_{49}$In </font>| 2n → <font color="Red">$^{189}_{85}$At </font>| ${\sigma}_{max}$ = 20 pb |
|$^{78}_{36}$Kr + $^{113}_{49}$In | 1n → $^{190}_{85}$At| ${\sigma}_{max}$ = 3 pb |
|$^{78}_{36}$Kr + $^{115}_{49}$In | 4n → $^{189}_{85}$At| ${\sigma}_{max}$ = 7 pb |
| <font color="Red"> $^{78}_{36}$Kr + $^{115}_{49}$In </font>| 3n → <font color="Red">$^{190}_{85}$At</font>| ${\sigma}_{max}$ = 232 pb |

</font>

***<mark style=background-color:Blue><font size=5 face="微软雅黑" color="Red" >New isotopes:projectile +target</font><mark>***

<font size=4 color="Red" face="微软雅黑"> 
   - $_{42}Mo + _{43}Tc$ → $^{190}_{85}$At